samza-yarn/src/test/java/org/apache/samza/job/yarn/TestFileSystemImplConfig.java

```diff
@@ -61,15 +57,28 @@ public void testNullConfig() {
   }
 
   @Test
-  public void testEmptyImpl() {
```

Comment:
It is because we removed getFsImplClassName(), and this test was originally for testing against getFsImplClassName().